.eslintrc.js

```diff
@@ -83,6 +85,12 @@ module.exports = {
         ...restrictedImports(),
       },
     },
+    {
+      files: ["e2e_testing/**/*.ts"],
+      settings: {
+        "disable/plugins": ["testing-library"],
```

Comment:
**Suggestion:** Rather than using an extra plugin, we could just handle this by removing the testing-library stuff from top config, and adding that in overrides:
```js
overrides: [
    {
      files: ["./frontends/**/*.test.{ts,tsx}"],
      plugins: ["testing-library"],
      extends: ["plugin:testing-library/react"],
    },
]
```